library/src/main/java/com/novoda/downloadmanager/ServiceNotificationDispatcher.java

```diff
@@ -62,25 +61,32 @@ void updateNotification(T payload) {
         };
     }
 
+    private void dismissStackedNotification(NotificationInformation notificationInformation) {
+        notificationManager.cancel(NOTIFICATION_TAG, notificationInformation.getId());
+    }
+
     private void updateNotification(NotificationInformation notificationInformation) {
```

Comment:
Maybe we can make it clearer in this method's name that it only deals with persistent notifications? Otherwise it may seem a bug that it always saves the ID without checking if it is a persistent notification. I had the doubt and had to read the rest of the class to understand how it's used.

Reply:
0f654ac